### PR TITLE
expand resolved tickets page; add created tickets page

### PR DESCRIPTION
**Pull request by Jamiras**
Replaces the `/user/X/tickets/resolved-for-others` page currently only accessible from the Developer Stats page with a more generic `/user/X/tickets/resolved` page with a "for others" filter that can be changed. This new page is also linked from the tickets list of the developer stats page.
https://discord.com/channels/476211979464343552/1002688256309211257/1274347936502780038

Additionally, adds a "Tickets Created" page to the account menu that allows a user to see any tickets they've created.
https://discord.com/channels/310192285306454017/962817919698501702/1259036308060508281

Both pages can be accessed for any user by changing the URL.

### CODE REVIEW

File: app/Platform/Services/TicketListService.php [hunk 1]
```diff
@@ -42,10 +44,17 @@ public function getFilterOptions(Request $request): array
...
     }
 
-    public function getSelectFilters(bool $showStatus = true, bool $showDevType = true, bool $showAchievementType = true): array
+    public function getSelectFilters(
+        bool $showStatus = true,
+        bool $showAchievementType = true,
+        bool $showDevType = true,
+        bool $showDeveloper = false,
+        bool $showReporter = false): array
     {
```
Comment (wescopeland) [inline app/Platform/Services/TicketListService.php:58]: nit: stylistically, don't we generally do
```php
): array {
```

**Review by wescopeland — APPROVED** [review]
Looks good, verified locally.

File: app/Platform/Services/TicketListService.php (hunk 1 above)
Comment (Jamiras) [reply app/Platform/Services/TicketListService.php:58]: Yep. I missed this when I changed it to multi-line and `composer lint` apparently doesn't care.

Comment (wescopeland) [reply app/Platform/Services/TicketListService.php:58]: No worries. I've also found how it handles whitespace can sometimes be unexpected or frustrating.

**Pull request merged**